lighthouse-core/gather/gather-runner.js

```diff
@@ -622,6 +622,18 @@ class GatherRunner {
     return !settings.disableStorageReset && passConfig.recordTrace && passConfig.useThrottling;
   }
 
+  /**
+   * Save the devtoolsLog and trace (if applicable) to baseArtifacts.
+   * @param {LH.Gatherer.PassContext} passContext
+   * @param {LH.Gatherer.LoadData} loadData
+   * @param {string} passName
+   */
+  static _saveLoadData(passContext, loadData, passName) {
```

Comment:
`save` makes me think of like a disk write for some reason, WDYT about 

`addLoadDataToBaseArtifacts`
`stashLoadDataOnBaseArtifacts`
`saveLoadDataToBaseArtifacts`
`augmentBaseArtifactsWithLoadData`

Reply:
`_addLoadDataToBaseArtifacts` 👍